### ISSUE TITLE
Dry-run on Kucoin?

### ISSUE THREAD

**Issue by edthepurple** [freqtrade/freqtrade#5646]
I can run it on binance with no problem but when I set it to Kucoin it doesn't start. log:

freqtrade    | 2021-10-01 06:29:40,665 - freqtrade.worker - INFO - Starting worker 2021.9
freqtrade    | 2021-10-01 06:29:40,665 - freqtrade.configuration.configuration - INFO - Using config: /freqtrade/user_data/config.json ...
freqtrade    | 2021-10-01 06:29:40,670 - freqtrade.commands.trade_commands - ERROR - Parse error at offset 899: Missing a comma or '}' after an object member.
freqtrade    | Please verify the following segment of your configuration:
freqtrade    |         "ccxt_config": {
freqtrade    |             "enableRateLimit": true
freqtrade    |             "rateLimit": 200
freqtrade    |         },
freqtrade    |         "ccxt_async_config": {
freqtrade    | 2021-10-01 06:29:40,670 - freqtrade.commands.trade_commands - ERROR - Fatal exception!
freqtrade    | Traceback (most recent call last):
freqtrade    |   File "/freqtrade/freqtrade/configuration/load_config.py", line 59, in load_config_file
freqtrade    |     config = rapidjson.load(file, parse_mode=CONFIG_PARSE_MODE)
freqtrade    | rapidjson.JSONDecodeError: Parse error at offset 899: Missing a comma or '}' after an object member.

**Comment by xmatthias** [comment]
your config is missing a comma after `"enableRateLimit": true`.

That whole segment should be:

```json 
"ccxt_config": {
        "enableRateLimit": true,
        "rateLimit": 200
    }
```

I've noticed it's wrong in the template - so it'll be fixed (in develop) in a second (you're faster fixing the config-file manually though - otherwise you'd have to regenerate the configuration).

**Comment by feydreva** [comment]
Issue is still present with fresh install / download as of today
Fix only in develop branch.

**Comment by edthepurple** [comment]
the docker image is not fixed either.

**Comment by xmatthias** [comment]
It's a small config typo that can easily be fixed in the configuration file (which usually has to be edited, or at least checked, anyway).

It'll be fixed with the next release (2021.10), and is fixed now in develop (as well as in develop docker images).
